Industry tooltips

phulsechinmay opened the pull request:
Closes #75 

Currently the tooltip is styled as a header with the offset highlighter effect. Can change it to normal text if we don't like it.

danielabreo commented:
when the tooltip appears, there's a hazy cloud surrounding it. can it be removed with css?

phulsechinmay commented:
@danielabreo I removed the circular box shadow and the highlighter effect. Also added a small box shadow on the tooltip to try and make it look slightly 3D, what do you think?

geooot commented:
@phulsechinmay 
I changed the bubble style to be a normal p element and adjusted the box-shadow to match the bubble. Also, the tooltip now appears at the top for the top bubble and on the bottom for the bottom bubble.
![image](https://user-images.githubusercontent.com/7832610/79888615-91b9fa00-83c2-11ea-9810-1c5339da967a.png)
